web/src/app/core/services/monitor.service.ts

```diff
@@ -0,0 +1,93 @@
+// Core components
+import { Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material';
+import { Router } from '@angular/router';
+
+// Firestore modules
+import { AngularFirestore } from '@angular/fire/firestore';
+import * as firebase from 'firebase';
+
+// Rxjs operators
+import { Observable } from 'rxjs';
+import { switchMap, take } from 'rxjs/operators';
+
+// Dashboard hub models and services
+import { MonitorModel, ProjectModel } from '../../shared/models/index.model';
+import { ActivityService } from './activity.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class MonitorService {
+
+  constructor(
+    private afs: AngularFirestore,
+    private activityService: ActivityService,
+    private snackBar: MatSnackBar,
+    private router: Router
+  ) {
+  }
+
+  /**
+   *
+   * @param monitors monitors list
+   * @param id id of monitor which needs to be find
+   * This function will return the monitor by id
```

Comment:
My understanding is the compodoc title is first then parameters.

```compodoc
/**
 * This is a good example
 * 
 * @param {string} target  The target to process see {@link Todo}
 * @returns The processed target number
 */
```

Reply:
done